packages/schematics/angular/utility/standalone/app_config.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Tree } from '@angular-devkit/schematics';
+import { dirname, join } from 'path';
+import ts from '../../third_party/github.com/Microsoft/TypeScript/lib/typescript';
+import { getSourceFile } from './util';
+
+/** App config that was resolved to its source node. */
+export interface ResolvedAppConfig {
+  /** Tree-relative path of the file containing the app config. */
+  filePath: string;
+
+  /** Node defining the app config. */
+  node: ts.ObjectLiteralExpression;
+}
+
```

Comment:
There's a bit of duplication here with the utilities we exposed for Components. The private utilities will be deleted once Components has been switched over.